src/PubSubCommunication.js

```diff
@@ -76,6 +76,7 @@ export default class PubSubCommunication {
     try {
       // Establishing pubsub connection. Compression is enabled by default.
       client = await pubSubClient.getClient();
+      process.env.PUBSUB_CONNECTION = client;
```

Comment:
is this also something that we can directly call from PubsubClient ?
Not depending on process.env for getting any status

Reply:
Basically, get the client details also via a function.

Reply:
Tried with importing pubsub client in LifecycleHistory.js but this will cause establishing LP connection again, we did not have better approach.